pkg/gofr/cron_test.go

```diff
@@ -606,3 +606,36 @@ func TestCron_parsePart(t *testing.T) {
 		})
 	}
 }
+
+func TestCron_WithTimezone(t *testing.T) {
+	est, _ := time.LoadLocation("America/New_York")
```

Comment:
It doesn't cost much

```suggestion
	est, err := time.LoadLocation("America/New_York")
	require.NoError(t, err, "computing prerequisite timezone")
```